coderd/workspaces.go

```diff
@@ -165,10 +165,37 @@ func (api *API) workspaceByOwnerAndName(rw http.ResponseWriter, r *http.Request)
 	owner := httpmw.UserParam(r)
 	workspaceName := chi.URLParam(r, "workspacename")
 
+	var (
+		includeDeletedStr = r.URL.Query().Get("include_deleted")
+		includeDeleted    = false
+	)
+	if includeDeletedStr != "" {
+		var err error
+		includeDeleted, err = strconv.ParseBool(includeDeletedStr)
```

Comment:
A nice trick in Go is to avoid unnecessary variables in the scope by defining them in the scope of the `if`.

```suggestion
	includeDeleted := false
	if s := r.URL.Query().Get("include_deleted"); s != "" {
		var err error
		includeDeleted, err = strconv.ParseBool(s)
```